indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -101,6 +101,16 @@
                    batch)]
     (es/prepare-batch context (filter-expired-concepts batch) options)))
 
+(defmethod prepare-batch :service
+  [context batch options]
+  ;; Get the service associations as well.
+  (let [batch (map (fn [concept]
+                     (let [service-associations (meta-db/get-associations-for-service
+                                                 context concept)]
+                       (assoc concept :variable-associations service-associations)))
```

Comment:
Should be `:service-associations`. Looks like we might be missing a test that verifies associations after bulk index.

Reply:
Fixed the `:service-associations`. Looking for where we have done any associations tests after bulk index ... have we done this before? Maybe I'm not grepping the right things.

Reply:
Ah, it seems that there's lots of implicit checking scattered around the sys-int tests for this (maybe that's not good enough? dunno). I had a bunch of failing tests for service associations after I made this change. With my misunderstandings addressed and related updates to my changes, all those failing tests now pass.